htslib/vcf.h

```diff
@@ -680,6 +683,7 @@ set to one of BCF_ERR* codes and must be checked before calling bcf_write().
      *  Returns 0 on success or negative value on error.
      */
     #define bcf_update_info_int32(hdr,line,key,values,n)   bcf_update_info((hdr),(line),(key),(values),(n),BCF_HT_INT)
+    #define bcf_update_info_int64(hdr,line,key,values,n)   bcf_update_info((hdr),(line),(key),(values),(n),BCF_HT_LONG)
```

Comment:
There should probably be a corresponding **get** macro at line 777:

```c 
#define bcf_get_info_int64(hdr,line,tag,dst,ndst)  bcf_get_info_values(hdr,line,tag,(void**)(dst),ndst,BCF_HT_LONG)

Reply:
Added.  I also decided to make `bcf_update_info_int64()` and `bcf_get_info_int64()` static inline functions instead of macros, so we can ensure that they are used with `int64_t` arrays.